lib/game/parallel.go

```diff
@@ -0,0 +1,251 @@
+package game
+
+import (
+	"sync"
+)
+
+type NeighborData struct {
+	Generation uint32
+	Data       []uint8
```

Comment:
nitpick: does not need to be now, but I'd give anything to avoid the dreaded `.Data.Data`

Reply:
Yeah, I also dislike that. Changed to `Cells`